frontend/src/css/PostRecipe.css

```diff
@@ -224,48 +33,29 @@ textarea {
   margin: 0 5px;
 }
 
-.form-content {
-  display: flex;
-  justify-content: space-between;
-  margin-top: 20px;
-}
-
-.input-subsection {
-  display: flex; /* Alinea los elementos secundarios en línea */
-  justify-content: space-between; /* Coloca elementos secundarios a los lados */
-  width: 100%; /* O ajusta el ancho según tus necesidades */
-}
-
-.ingredients-section, .preparation-section {
-  width: 48%; /* Cambia el ancho según tus necesidades y agrega un margen si es necesario */
-  box-sizing: border-box; /* Evita que el margen y el relleno afecten el ancho total */
-  padding: 10px; /* Añade relleno según tus preferencias */
-  border-radius: 10px; /* Agrega bordes redondeados si es necesario */
-}
-
-
-.ingredient-item, .preparation-item {
-  display: flex;
-  justify-content: space-between;
-  padding: 5px;
-}
-
-.ingredient-input-group, .instruction-input-group {
-  margin-top: 1px;
-}
-
-.ingredient-unit-select{
+.ingredient-unit-select {
```

Comment:
This can be done through bootstrap classes. Maybe 1 pixel is not achievable, but at this point is a stylesheet really necessary? For a single value the style attribute could be used directly in the component.